libs/ui-watt/project.json

```diff
@@ -40,26 +37,17 @@
       }
     },
     "build-storybook": {
-      "executor": "@nrwl/storybook:build",
+      "executor": "@storybook/angular:build-storybook",
       "outputs": ["{options.outputPath}"],
       "options": {
-        "outputPath": "dist/designsystem",
-        "projectBuildConfig": "ui-watt:build-storybook",
-        "uiFramework": "@storybook/angular",
-        "config": {
-          "configFolder": "libs/ui-watt/.storybook"
-        },
-        "assets": [
-          {
-            "input": "libs/ui-watt/src/assets",
-            "glob": "**/*",
-            "output": "assets/ui-watt"
-          }
-        ],
         "styles": ["libs/ui-watt/.storybook/styles.scss"],
         "stylePreprocessorOptions": {
           "includePaths": ["libs/ui-watt/src/lib/styles"]
-        }
+        },
+        "outputDir": "dist/designsystem",
+        "configDir": "libs/ui-watt/.storybook",
+        "browserTarget": "ui-watt:build-storybook",
+        "compodoc": false
```

Comment:
Could this option actually let us get rid of our custom compodoc script (docs:json) setting this to true?

Reply:
Good question. Haven't tried it. Will do and write back whether it's possible. It'd be nice it if is :)